### PR TITLE
domd: Compile netevent in one thread

### PR DESCRIPTION
**Pull request by rshym**
Sometimes we observe race during multithreaded compilation
so it's better to build netevent in one thread.

### CODE REVIEW

**Comment by rshym** [comment]
Patch tested by build of DomD in following configurations:
- for `h3ulcb-4x2g-kf` without patch - observed 'make -j 8' in domd's `log.do_compile`
- for `h3ulcb-4x2g-kf` with patch - observed 'make' in domd's `log.do_compile`
- for `salvator-xs-h3-4x2g` with patch - checked that no 'No recipes available for' warning is generated

File: recipes-domd/domd-image-weston/domd-image-weston.bbappend [hunk 1]
```diff
@@ -175,6 +175,13 @@ configure_versions_rcar() {
     fi
 
     base_update_conf_value ${local_conf} XT_RCAR_PROPRIETARY_MULTIMEDIA_DIR "${XT_RCAR_PROPRIETARY_MULTIMEDIA_DIR}"
+
+    # We have netevent_%.bbappend that modifies
+    # meta-rcar-gen3-adas/recipes-support/netevent/netevent_git.bb
+    # and is intended to be used for Kingfisher only.
+    # For other boards we need to mask our bbappend to avoid
+    # warning "No recipes available for".
+    base_add_conf_value ${local_conf} BBMASK "recipes-support/netevent"
```
Comment (lorc) [inline recipes-domd/domd-image-weston/domd-image-weston.bbappend:184]: This is a good change. But it should be made in a separate commit.

Comment (rshym) [reply recipes-domd/domd-image-weston/domd-image-weston.bbappend:184]: Done. BBMASK is moved to separate commit.

**Comment by lorc** [comment]
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by rshym** [comment]
Reviewer is added.

**Review by iusyk — COMMENTED** [review]
A proposal - create a separate template for king_fisher.local.conf  and put BBMASK there. It easier to find the source of BBMASK

**Comment by rshym** [comment]
> A proposal - create a separate template for king_fisher.local.conf and put BBMASK there. It easier to find the source of BBMASK

Unfortunately, this will not work, because BBMASK need to be used for all boards except Kingfisher.
